types/react-virtualized-select/index.d.ts

```diff
@@ -24,7 +24,7 @@ export interface VirtualizedOptionRenderOptions<T> {
 
 export interface AdditionalVirtualizedSelectProps<TValue> {
     maxHeight?: number;
-    optionHeight?: number;
+    optionHeight?: number | ((option: TValue) => number);
```

Comment:
Should this actually be `({ option: TValue }) => number`? The docs you linked suggest so, but I'm not sure what `TValue` generally is (is it an object with an `option` field?) as I haven't used this library in years.

Reply:
Yes, you're right. Good catch, thanks! Fixed it.

Reply:
😕but now the build is failing after that change. Not really sure what to do.

Reply:
> Binding element 'TValue' implicitly has an 'any' type

I guess it's asking to name the `{ option: TValue }` type? Maybe you could introduce `interface WrappedValue<TValue>` and use that instead of an inline type?

Reply:
I just ran into this same pattern in my own code. The problem is just that it wants an argument name: `(options: { option: TValue }) => ...`.

Reply:
Thanks @seansfkelley! I went ahead and made that change.